test/python/compiler/test_assembler.py

```diff
@@ -629,20 +629,16 @@ def test_pulse_name_conflicts_in_other_schedule(self):
         self.assertEqual(len(qobj.config.pulse_library), 3)
 
     def test_assemble_with_delay(self):
-        """Test that delay instruction is ignored in assembly."""
+        """Test that delay instruction in assembly."""
         backend = FakeOpenPulse2Q()
+        delay_schedule = pulse.Delay(10)(self.backend_config.drive(0))
```

Comment:
either add back `self.assertWarns(DeprecationWarning):` or change this to `pulse.Delay(10, self.backend_config.drive(0))`

Reply:
Fixed in [5d77a75](https://github.com/Qiskit/qiskit-terra/pull/4233/commits/5d77a757fecea39889b10b876b0e472127520e41).